src/GridList.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module GridList
+ */
+class GridList extends React.Component {
+	renderItems() {
+		this.gridListItems = this.props.items.map((item, i) => {
+			return (
+				<div
+					key={i}
+					className='gridList-item'
+				>
+					{item}
+				</div>
+			);
+		});
+		return this.gridListItems;
+	}
+	render() {
+		const {
+			className,
+			columns,
+			items, // eslint-disable-line no-unused-vars
+			...other
+		} = this.props;
+
+		const classNames = cx(
+			'gridList',
+			{
+				[`gridList--has${columns.default}`]: typeof columns.default === 'number',
```

Comment:
prefer to keep your 'type-enforcement' code in one place - since `number` is already 'enforced' by the proptypes, I recommend just checking for the existence of the prop rather than the type here

```js
[`gridList--has${columns.default}`]: columns.default  // or Boolean(columns.default)
```